Remotion/Web/Development.WebTesting.IntegrationTests/ListMenuControlObjectTest.cs

```diff
@@ -154,6 +155,36 @@ public void TestClickItem ()
       Assert.That(home.Scope.FindId("TestOutputLabel").Text, Is.EqualTo("ItemID4|WxeFunction"));
     }
 
+    [Test]
+    public void TestHeading ()
+    {
+      var home = Start();
+
+      var listMenu = home.ListMenus().GetByLocalID("MyListMenu");
+      var heading = listMenu.Heading;
+
+      Assert.That(heading, Is.EqualTo("Test List Menu"));
+    }
+
+    [Test]
+    public void TestMissingHeading ()
+    {
+      var home = Start();
+
+      var listMenu = home.ListMenus().GetByLocalID("MyListMenu2");
+
+      var initialTimeout = listMenu.Scope.ElementFinder.Options.Timeout;
```

Comment:
please update the controlobject to first test if the heading exists (e.g. by introducing a `hasHeading` attribute or by testing for the `labelledby` Attribute (if this works). The goal is to not have a timeout whenever the usecase isn't an error case.